.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+  - repo: https://github.com/kynan/nbstripout
+    rev: 0.6.0
+    hooks:
+      - id: nbstripout
+        args: [--extra-keys="cell.metadata.pycharm" --strip-empty-cells]
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        args: ["--profile", "black", "--line-length=120"]

```

Comment:
I've discussed 120-line length for import statements with @Raalsky  and we've decided to try to configure `isort` for preferring multi-line imports and forcing black not to complain about that.
[here's](https://github.com/neptune-ai/neptune-client/pull/1031#discussion_r988029355) the discussion.